test/testdata/generate.sh

```diff
@@ -69,6 +69,10 @@ function scaffold_test_project {
       $kb create webhook --group crew --version v1 --kind Admiral --defaulting
     fi
 
+    if [ $project == "project-v3-with-deploy-image" ]; then
+      $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false
```

Comment:
```suggestion
      $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true
      $make manifests
```
Can we check with? 
In this case, we probably want to run make manifests to see the rules and etc.